tgui/packages/tgui/interfaces/GlassRecycler.js

```diff
@@ -0,0 +1,103 @@
+import { Fragment } from 'inferno';
+import { useBackend, useLocalState } from "../backend";
+import { Box, Button, Divider, Flex, Section, Stack } from "../components";
+import { Window } from "../layouts";
+
+// This already exists in WeaponVendor/index.tsx, but feels weird
+// to make it a dependency.
+const pluralize = (word, n) => (n !== 1 ? word + 's' : word);
+const capitalize = (word) => word.replace(/(^\w{1})|(\s+\w{1})/g, letter => letter.toUpperCase());
```

Comment:
should be abstracted out to a helper file for both then

Reply:
There a specific place I should be making this util file? Is placing it in `/tgui/packages/tgui` fine?

Reply:
Refactored into helper file `stringUtils`.